orchestrator/api/fleet.py

```
@@ -0,0 +1,42 @@
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/api/v1/fleet", tags=["fleet"])
+
+# The board plane's read guard, mirrored (PRD-228 §7 / api/board_tasks.py).
+_require_fleet_read = require_task_context(TASKS_READ)
+
+
+@router.get("")
+async def get_fleet(
+    ctx: RequestContext = Depends(_require_fleet_read),
```

Ruff (0.16.2)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)